src/core/containers/HandleLogin/index.js

```diff
@@ -1,94 +0,0 @@
-import React from 'react';
-import PropTypes from 'prop-types';
-import cookie from 'react-cookie';
-import { connect } from 'react-redux';
-import { compose } from 'redux';
-import config from 'config';
-import { withRouter } from 'react-router';
-
-import { setAuthToken } from 'core/actions';
-import { login } from 'core/api';
-import LoginPage from 'core/components/LoginPage';
-import log from 'core/logger';
-import { browserBase64Decode, gettext as _ } from 'core/utils';
```

Comment:
It looks like you can delete `browserBase64Decode` too.

Reply:
thanks!

Reply:
It didn't have any tests either, heh